### DISCUSSION TITLE
isn't "natively" a word?

### DISCUSSION THREAD

**Discussion by anarcat** [Automattic/harper#171] [Ideas]
harper marks "natively" as not a word. It seems to me a valid word... I considered sending a PR to https://github.com/elijah-potter/harper/blob/master/harper-core/dictionary.dict but (a) i'm not sure it's the right place and (b) i'm not sure of the syntax of that file... where is it from?

i'm not sure of my english anymore either. ;)

**Comment by elijah-potter** [comment]
You're absolutely right that "natively" is a word. I'll add it to the bundled dictionary. Feel free to reopen this discussion if there are any other words that should be added.

**Reply by anarcat** [reply]
i actually can`t figure out how to reopen this, i'll file an issue for `ipv4` that's getting flagged.

**Reply by anarcat** [reply]
i filed a bunch more issues, actually. note that you can reproduce those issues by running harper against this document:

https://gitlab.torproject.org/tpo/tpa/wiki-replica/-/blob/84d70d2822392c1731013f526b7b2c182db10004/service/prometheus.md?plain=1

you can also see the last round of corrections i've done with Harper, including some that i consider to be workaround for acceptable things, in this commit:

https://gitlab.torproject.org/tpo/tpa/wiki-replica/-/commit/84d70d2822392c1731013f526b7b2c182db10004

In particular, I'm struggling a bit with GitLab short links like `[tpo/tpa/team#41070][]` and kicking myself for starting paragraphs with:

```
.... and that's what happened.
```

but maybe it's for the best?
